### ISSUE TITLE
4chan (is2.4chan.org) issues

### ISSUE THREAD

**Issue by canyalccn** [ValdikSS/GoodbyeDPI#360]
### Operating system / операционная система

Windows 10 22H2

### Running as service / Запуск программы как сервис

I run it as a regular program / Запускаю программу обычным образом

### Describe the bug / Опишите ошибку программы

4chan.org works perfectly, but whenever I try to click a image to get it bigger, it doesn't work. I discovered that it only happens with pics linked with is2.4chan.org, which is a site for holding images i guess. It works when i right click the pic and open it in a new tab which directs me to i.4cdn.org (but it's the compressed version of the pic). same with videos. other than that specific site everything works flawlessly with goodbyedpi.

### Additional information / Дополнительная информация

-changing browsers or turning off kyber didnt do anything
-I was using an older version of goodbyedpi with gui it got working after 5-10 clicks or reloads on that specific site. then changed to latest build (22), worked good for 1-2 months now it doesn't open any images or videos at all.

there is certainly some pattern here but i can't see it. anyone experiencing the same issue or manage to fix it?

**Comment by canyalccn** [comment]
I checked edge, and it works after few tries. still curious about what causes this though

**Comment by UserAlreadyDead** [comment]
Yep, i got the same problem with 4chan recently. Don't know how to solve this yet though.

**Comment by ValdikSS** [comment]
This doesn't seem like a bug report for me. If you have any other information, feel free to add it.

**Comment by canyalccn** [comment]
> Yep, i got the same problem with 4chan recently. Don't know how to solve this yet though.

I tweaked the fragment values yesterday. It started working stably after increasing the values to 10 from 2, for both HTTP and HTTPS. Though 10 is way too much, I guess; Chrome wasn't very stable with 5, so trying a middle ground could help maybe. I don't know how much these values would affect CPU usage and speed in general. I'm not very savvy about all this, so extra info from a more knowledgeable person would be nice. Also, it's compatibility changes on different browsers as well. Edge and Firefox needed less fragmentation to get the site working compared to Chrome, but it's probably a personal thing; I can't assure it would be the same for everyone else. Changing to other modesets didn't help either.

Another thing I tried is adding the IP ID of is2.4chan.org to .exe using the --ip-id parameter. I wasn't expecting it to work since the IPs of every image/video link should be different from each other, but surprisingly, it worked, at least for a while. Then back to normal, I don't know what happened there. If you wanna try out yourself the IP IDs of is2.4chan.org are:
--ip-id 158.69.18.72 --ip-id 144.217.129.129 --ip-id 144.217.123.248 --ip-id 142.44.218.139

What boggles my mind is what causes it to stop working after working perfectly fine for 1-2 months. The same happened when using older versions as well.
